src/components/QueuedTransactionList/index.tsx

```diff
@@ -0,0 +1,167 @@
+import { Identicon, Text } from '@gnosis.pm/safe-react-components'
+import {
+  getTransactionQueue,
+  Transaction,
+  TransactionListItem,
+  TransactionSummary,
+} from '@gnosis.pm/safe-react-gateway-sdk'
+import { List } from '@material-ui/core'
+import { ReactElement, useEffect, useState } from 'react'
+import { useSelector } from 'react-redux'
+
+import { CircleDot } from 'src/components/AppLayout/Header/components/CircleDot'
+import PrefixedEthHashInfo from 'src/components/PrefixedEthHashInfo'
+import { getChainById } from 'src/config'
+import { ChainId } from 'src/config/chain.d'
+import useOwnerSafes from 'src/logic/safe/hooks/useOwnerSafes'
+import { userAccountSelector } from 'src/logic/wallets/store/selectors'
+import { black300, grey400 } from 'src/theme/variables'
+import { checksumAddress } from 'src/utils/checksumAddress'
+import { GATEWAY_URL } from 'src/utils/constants'
+
+import styled from 'styled-components'
+
+export const isTransactionType = (value: TransactionListItem): value is Transaction => {
+  return value.type === 'TRANSACTION'
+}
+
+const ChevronRight = (
```

Comment:
Does MUI icons not have this already?

Reply:
It does indeed! Good one! I only checked SRC.